src/styles/components/Button.scss

```diff
@@ -25,46 +25,153 @@
         border-color: $btn-default;
         size: $btn-size;
         height: $btn-height;
+        box-shadow: none !important;
+
+        &:focus{
+            outline: none !important;
+            border: $blue-450 $border-thick double;
+        }
+    
+        &:hover{     
+            border-color: $btn-hover;
+        }
+    
+        &:active{
+            border: $blue-600;
```

Comment:
Please declare button-specific color code in _colors.scss
btn-hover-color etc.

Reply:
Updated for hover as the color was same, but for active and other actions the color can vary based on button type. So instead of making variable for each which will be specific for that button, I am directly using the color variable.